apps/gnome-back/src/gnomes/gnomes.service.ts

```diff
@@ -138,6 +138,10 @@ export class GnomesService {
     return {
       ...createGnome,
       gnome: findGnome,
+      resources: {
+        [resource1]: updatedResources[resource1],
+        [resource2]: updatedResources[resource2],
+      },
```

Comment:
myślałem tutaj bardziej że będą zwracane zasoby które dostałeś z tego konkretnego krasnala ale to co podrzuciłeś też jest fajne

zerknij czy w response nie trzeba tego dodać też w shared

a najlepiej jakby to wszystko było zwracane w taki sposób:


```{
...,  
gnome: ...,
_metadata: {
  user_resources: {
    // aktualne wszystkie resource usera
  },
  gathered_resources: {
    // świeżo zdobyte resource
  }
}  
}```